packages/chat-component-bindings/src/chatParticipantListSelector.ts

```diff
@@ -5,24 +5,28 @@ import { getUserId, getDisplayName, getParticipants, ChatBaseSelectorProps } fro
 import * as reselect from 'reselect';
 import { toFlatCommunicationIdentifier } from '@internal/acs-ui-common';
 import { ChatParticipant } from '@azure/communication-chat';
-import { CommunicationParticipant } from '@internal/react-components';
+import { ParticipantListParticipant } from '@internal/react-components';
 import { ChatClientState } from '@internal/chat-stateful-client';
+import { getIdentifierKind } from '@azure/communication-common';
 
 const convertChatParticipantsToCommunicationParticipants = (
   chatParticipants: ChatParticipant[]
-): CommunicationParticipant[] => {
+): ParticipantListParticipant[] => {
   return chatParticipants.map((participant: ChatParticipant) => {
     return {
       userId: toFlatCommunicationIdentifier(participant.id),
-      displayName: participant.displayName
+      displayName: participant.displayName,
+      // ACS users can not remove Teams users.
```

Comment:
Havn't really tested this because there is no easy way to join a Teams thread without using the MeetingComposite. There, we use the Calling roster ignoring the chat one.